### PR TITLE
Added PPP

### LINKED ISSUE #270: User Guide: minor errors
- User Guide's `exit` clicking of the "x" does not include macOS style window managers.

- Sample currency.json is not given

### PR DESCRIPTION
**Pull request by krusagiz**
I have added my _Project Portfolio Page_.

Also resolves #270

### CODE REVIEW

File: docs/team/krusagiz.adoc [hunk 1]
```diff
@@ -1,7 +1,166 @@
...
+
+=== Code contributed
+You can follow these links to view a sample of my code: +
+ [https://github.com/AY1920S1-CS2103T-W12-2/main/commits/master?author=krusagiz[Commits]] [https://github.com/AY1920S1-CS2103T-W12-2/main/pulls?utf8=%E2%9C%93&q=is%3Amerged+is%3Apr+author%3Akrusagiz+[Pull requests]] [https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=krusagiz&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false[RepoSense Dashboard]]
+
+=== Functionality enhanced
+
+The following are some of my enhancements:
+
+* **Major enhancement:** Added the **ability to tag and un-tag** in transaction list.
```
Comment (lightz96) [inline docs/team/krusagiz.adoc]: ... in **the** transaction list.

File: docs/team/krusagiz.adoc [hunk 2]
```diff
@@ -1,7 +1,166 @@
...
+** Justification: Tags in THRIFT are meant to categorise the different transactions, it is more meaningful if tags are of different colour if they correspond to different categories.
+** Highlights: The colours are coloured into different colours if they are a <<../UserGuide#TagColours, keyword>>, otherwise they are in Purple.
+
+
+=== Other contributions
+* Project management:
+** I managed issues assigned to me from version 1.0 to 1.4 on https://github.com/AY1920S1-CS2103T-W12-2/main/milestones[GitHub].
+
+* Enhancement to existing features:
+** I added a function to to the existing class `**ArgumentMultimap**` to enforce the return of no more than one argument value. (Pull request https://github.com/AY1920S1-CS2103T-W12-2/main/pull/118[#118])
```
Comment (lightz96) [inline docs/team/krusagiz.adoc]: I added a function **to** the existing class...

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -456,21 +456,21 @@ since `**Budget**` is not really related to `**Transaction**` in the real world.
...
 
 .A class diagram illustrating the implementation of `**Tag**` inside a `**Transaction**`
-image::tag-untag/TagPolymorphism.png[]
+image::tag-untag/TagPolymorphism.png[,250]
 
-Due to polymorphic nature of `**Transaction**`, many of the driver functions in `**THRIFT**` uses the `**Transaction**` class for both `**Expense**`
+Due to this polymorphic implementation, many of the driver functions in `**THRIFT**` use the `**Transaction**` class for both `**Expense**`
 and `**Income**` objects. For example, when tagging a new `**Expense**`/`**Income**` object, the `**TagCommandParser**` will treat them as the same object:
 
 .An sequence diagram showing how the `**TagCommandParser**` is called
```
Comment (lightz96) [inline docs/DeveloperGuide.adoc]: **A** sequence diagram

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -456,21 +456,21 @@ since `**Budget**` is not really related to `**Transaction**` in the real world.
...
 === Tag/Untag feature
 
-We allow the user to add and remove custom `**Tag**` objects in `**Transaction**` entries so that they can categorise and partition the entries to their own liking.
+We allow the user to add and remove custom `**Tag**` objects in `**Transaction**` entries so that they can categorise the entries to their own liking.
 The following sections describe how this feature is implemented and the design considerations that led up to the solution.
 
 ==== Implementation
 
-The **tag**/**untag** command performs modifications on existing `**Transaction**` entries. Due to polymorphism, both `**Expense**` and `**Income**` objects extends the abstract class `**Transaction**`, and are treated the same way in the context of the **tag**/**untag** command.
+The **tag**/**untag** command performs modifications on existing `**Transaction**` entries which reside in the `**Model**`. Implemented using the concept of polymorphism , both `**Expense**` and `**Income**` objects extend the abstract class `**Transaction**`, and are treated the same way in the context of the **tag**/**untag** command.
```
Comment (lightz96) [inline docs/DeveloperGuide.adoc]: entries **that reside** in the `**Model**`

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -512,12 +512,12 @@ Tags that do not exist inside the specified `**Transaction**` will be ignored, a
 
 Again, since the user input is valid, a `**UntagCommand**` command object is created and executed. As a result, a copy of the `**Transaction**` object with the specified `**Tag**` deleted replaces the original in the `**TransactionList**` at the same position.
 
-image::tag-untag/TagUntag1.png[]
+image::tag-untag/TagUntag1.png[,450]
 
-The following activity diagrams summarises what happens when the user executes a command to **tag** a `**Transaction**`.
+The following activity diagrams summarises what happens when the user executes a command to tag a `**Transaction**`.
```
Comment (lightz96) [inline docs/DeveloperGuide.adoc]: The following activity diagrams **summarise** what happens...

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -998,7 +994,6 @@ due to its key size.
 
 We believe that both alternatives have reasonable pros and cons and choosing either option would be beneficial to `**THRIFT**`.
 The added data security will assure the user that their data is safe and allows them to use `**THRIFT**` with a peace of mind.
-
 // end::dataencryption[]
```
Comment (lightz96) [inline docs/DeveloperGuide.adoc:998]: Under **Aspect: Uniqueness of Tag objects within a Transaction** alternative 1's cons:

... as they are **no** longer...

**Review by lightz96 — CHANGES_REQUESTED** [review]
LGTM after fixing some typo errors.

File: docs/team/krusagiz.adoc (hunk 1 above)
Comment (krusagiz) [reply docs/team/krusagiz.adoc]: Fixed.

File: docs/team/krusagiz.adoc (hunk 2 above)
Comment (krusagiz) [reply docs/team/krusagiz.adoc]: Fixed.

File: docs/DeveloperGuide.adoc (hunk 4 above)
Comment (krusagiz) [reply docs/DeveloperGuide.adoc]: Fixed.

File: docs/DeveloperGuide.adoc (hunk 3 above)
Comment (krusagiz) [reply docs/DeveloperGuide.adoc]: Fixed.

File: docs/DeveloperGuide.adoc (hunk 5 above)
Comment (krusagiz) [reply docs/DeveloperGuide.adoc]: Fixed.

File: docs/DeveloperGuide.adoc (hunk 6 above)
Comment (krusagiz) [reply docs/DeveloperGuide.adoc:998]: Fixed.

**Review by lightz96 — APPROVED** [review]

**Pull request merged**